seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java

```diff
@@ -73,6 +73,24 @@ protected final void prepare(E env, List<? extends Plugin<E>>... plugins) {
         }
     }
 
+    /**
+     * Execute close method defined in {@link org.apache.seatunnel.plugin.Plugin}
+     *
+     * @param plugins plugin list
+     */
+    @SafeVarargs
+    protected final void close(List<? extends Plugin<E>>... plugins) {
+        for (List<? extends Plugin<E>> pluginList : plugins) {
+            pluginList.forEach(plugin -> {
+                try (Plugin<?> closed = plugin) {
```

Comment:
can you add a failure case UT? it seems these code can not achive the 'close all' goal.

Reply:
Oh yes, @ruanwenjun  you still throw the exception once there is a plugin throws an exception, causing the remaining plugins not to closed